src/platforms/apple/common/dsym.mdx

```diff
@@ -24,9 +24,13 @@ _NOTE_: This feature will be available as an early access preview soon.
 
 When you use the App Store Connect integration, Sentry will automatically discover and fetch dSYM files for Bitcode builds as they become available on App Store Connect. These dSYM files however contain obfuscated symbol names and paths. An additional BCSymbolMap file needs to be uploaded to provide properly symbolicated crash reports. These BCSymbolMap files can be uploaded using either [sentry-cli](#symbolmap-sentrycli) or [Fastlane](#symbolmap-fastlane).
 
-To configure the integration, go to **[Project] > Settings > Debug Files** and add App Store Connect as a new "Custom Repository". Note that setting up the integration will currently result in two identical audit entries being created.
+To configure the integration, go to **[Project] > Settings > Debug Files** and add App Store Connect as a new "Custom Repository".
 
-Once setup the integration will ensure that new builds are detected and fetched within an hour of being available on App Store Connect.
+![Getting to a project's settings](getting-to-project-settings.png)
```

Comment:
Getting to the project settings is a pretty straightforward direction and adding this particular screenshot doesn't add more information for the reader, so I'd remove this one.  
```suggestion
```